controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/DeploymentUpgrader.java

```diff
@@ -46,7 +46,6 @@ protected double maintain() {
                         Run last = controller().jobController().last(job).get();
                         Versions target = new Versions(systemVersion, last.versions().targetApplication(), Optional.of(last.versions().targetPlatform()), Optional.of(last.versions().targetApplication()));
                         if ( ! deployment.version().isBefore(target.targetPlatform())) continue;
-                        if (   controller().clock().instant().isBefore(last.start().plus(Duration.ofDays(1)))) continue;
                         if ( ! isLikelyNightFor(job)) continue;
```

Comment:
I guess someone could theoretically block upgrade by redeploying once an hour, or so, but until someone actually does this, let's leave this. The remedy for that would be to always upgrade if the version is more than, say, 7 days old. Then again, that someone could deploy with a particular runtime version once an hour ... and this is already possible today, so I don't think it's actually a problem.